---
std_msgs/Bool is_ok
geometry_msgs/Pose target_pose